tools/toolchains/arm.py

```diff
@@ -81,6 +81,12 @@ def __init__(self, target, notify=None, macros=None,
             if "--library_type=microlib" not in self.flags['common']:
                 self.flags['common'].append("--library_type=microlib")
 
+        if (
+            target.printf_lib == "minimal-printf" and
```

Comment:
I'd prefer these two lines joined together but that's a minor point.

Reply:
It makes the line too long not very Pythonic. It's sorta standard to keep lines under 79 chars https://black.readthedocs.io/en/stable/the_black_code_style.html#line-length

Reply:
I meant Lines 84 & 85

Reply:
The popular formatter "Black" would format it more or less as I did.
```bash
$ black -l 79 test_python.py --diff
reformatted test_python.py
All done! ✨ 🍰 ✨
--- test_python.py      2019-11-14 14:51:46.563480 +0000
+++ test_python.py      2019-11-14 14:51:52.001901 +0000
@@ -1,2 +1,6 @@
-if target.printf_lib == "minimal-printf" and "-DMBED_MINIMAL_PRINTF" not in self.flags['common']:
+if (
+    target.printf_lib == "minimal-printf"
+    and "-DMBED_MINIMAL_PRINTF" not in self.flags["common"]
+):
     pass
```